app/index.html

```diff
@@ -24,6 +24,7 @@
     <link rel="stylesheet" href="bower_components/kubernetes-container-terminal/dist/container-terminal.css" />
     <link rel="stylesheet" href="bower_components/registry-image-widgets/dist/image-widgets.css" />
     <link rel="stylesheet" href="bower_components/layout.attrs/dist/layout.attrs.css" />
+    <link rel="stylesheet" href="bower_components/bootstrap-slider/dist/css/bootstrap-slider.css" />
```

Comment:
This vendor CSS isn't necessary as it is included in PatternFly, so add `bower_components/bootstrap-slider/dist/css/bootstrap-slider.css` to the Gruntfile ignore.

Reply:
Fixed, thanks.  I'll keep an eye on 'bower_components/bootstrap-slider/dist/bootstrap-slider.js', it shouldn't be included either but waiting for patternfly core version release which includes bootstrap-slider in bower